frontend/src/MuiForms/TagSelector.tsx

```diff
@@ -77,16 +79,27 @@ export default function TagSelector({ onChange, value, label, formContext, requi
             </Stack>
             <Box sx={{ overflowX: 'auto', p: 1 }}>
               <Stack spacing={1} direction='row'>
-                <Box sx={{ whitespace: 'pre-wrap' }}>
-                  {value.map((tag) => (
-                    <Chip
-                      label={tag}
-                      key={tag}
-                      sx={{ width: 'fit-content', m: 0.5 }}
-                      onDelete={() => handleChipOnDelete(tag)}
-                    />
-                  ))}
-                </Box>
+                {value.length === 0 ? (
+                  <Typography
+                    sx={{
+                      fontStyle: 'italic',
+                      color: theme.palette.customTextInput.main,
+                    }}
+                  >
+                    No tags
```

Comment:
I've changed my mind on this... because this _could_ be used in the future by the JSON schema it might be worth replacing this with:

`formContext && formContext.emptyPlaceholderText ? formContext.emptyPlaceholderText : 'No tags'`

That way, if we want to use it for forms we can set the text that way, as the question its used for might not be called "tags" but still use this component.